pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554</version>
+    <version>1.580</version>
```

Comment:
My guess is there would be a ton of work to adapt to the new plugin pom, at least more than necessary and should be in a separate change in that case.